webpack/assets/javascripts/react_app/components/notifications/index.js

```diff
@@ -33,11 +33,12 @@ class notificationContainer extends React.Component {
   }
 
   componentDidUpdate() {
-    const { error } = this.props;
+    const { error, stopNotificationsPolling } = this.props;
     if (error) {
       const { response: { status } = {} } = error;
       if (status === 401) {
-        redirectToLogin();
+        stopNotificationsPolling();
```

Comment:
We should stop not only on 401, but on all errors

Reply:
Are you sure? HTTP 502/503 can be a temporary error due to some startup.

Reply:
I think it would be better error on the side of stopping the polling than continue trying and possibly hammer the server with requests every 10 seconds